### ISSUE TITLE
Add chip after Enter key or allow to add chips programmatically

### ISSUE THREAD

**Issue by oliverbytes** [zsoerenm/chips_input#20]
Like how most popular apps implement them. Pressing the enter key / submitting automatically adds the entered text as a chip.

**Comment by zsoerenm** [comment]
How should that work?
What if the entered text does not match a suggestion, or what if the entered text matches multiple suggestions?

**Comment by oliverbytes** [comment]
> How should that work?
> What if the entered text does not match a suggestion, or what if the entered text matches multiple suggestions?

Yeah. I think If it doesn't match a suggestion, use it. If matches multiple suggestions, use the first one.

**Comment by zsoerenm** [comment]
Your first suggestion won't work, if the suggestions are objects and not strings.
I think this is not general enough. You could implement an `onSubmitted` or `onEditingComplete` callback to do what you want. The `onSubmitted` isn't implemented, yet. I am open for pull requests, though.

**Comment by huertin03** [comment]
I can see oliverbytes point. I'm also into it. Trying to achieve it `onEditingComplete` works vaguely for me as I can't get the Chip to be created
